tests/acceptance/bootstrap/SettingsContext.php

```diff
@@ -596,11 +597,109 @@ public function getBodyForNotificationSetting(string $user): array {
 						"resource" => [
 							"type" => $settingsValue->resource->type
 						],
-						"boolValue" => true
 					]
 				];
 			}
 		}
-		throw new Exception(('`disable-email-notifications` not found in the setting list'));
+		throw new Exception(("'$event' not found in the setting list"));
```

Comment:
do we need double bracket (( ? 
```suggestion
		throw new Exception(("'$event' not found in the setting list"));
```